scripts/resource/RoomData.gd

```diff
@@ -1,24 +1,26 @@
-class_name Room extends Resource
+class_name RoomData extends Resource
 
-@export var id: int
-@export var room_type: RoomType
+var id: int
+@export var type: RoomType
 @export var top_left: Vector2i
 @export var bottom_right: Vector2i
 @export var door_tiles: Array[Vector2i]
 @export var hot_spots: Array[Vector2i]
+var assigned_crew_ids: Array[int]
 
 func _init(
-	p_id: int = 0,
-	p_room_type: RoomType = RoomType.new(), # Instantiate a blank RoomType
-	p_top_left: Vector2i = Vector2i(0, 0),
-	p_bottom_right: Vector2i = Vector2i(0, 0)
+	room_id: int = 0,
+	room_type: RoomType = RoomType.new(), # Instantiate a blank RoomType
+	room_top_left: Vector2i = Vector2i(0, 0),
+	room_bottom_right: Vector2i = Vector2i(0, 0)
 ):
-	id = p_id
-	room_type = p_room_type
-	top_left = p_top_left
-	bottom_right = p_bottom_right
+	id = room_id
+	room_type = room_type
```

Comment:
Do we need self.room_type on the left if the names match?

Reply:
you're right i missed updating the name for this one